apps/docs/docs/cdk/render-strategies/render-strategies.mdx

```diff
@@ -174,77 +174,65 @@ By default the following configurations are set:
 
 ### Global
 
-1. Module Import
+1. Configure in app root configuration
 
 ```typescript
-...
-import {RxRenderStrategiesConfig, RX_RENDER_STRATEGIES_CONFIG} from '@rx-angular/cdk/render-strategies';
+import { provideRxRenderStrategies } from '@rx-angular/cdk/render-strategies';
 
-const CUSTOM_RX_ANGULAR_CONFIG: RxRenderStrategiesConfig<string> = {
-  primaryStrategy: 'global',
-  patchZone: false
-}
-
-@Module({
+const appConfig: ApplicationConfig = {
   providers: [
-    {
-      provide: RX_RENDER_STRATEGIES_CONFIG,
-      useValue: CUSTOM_RX_ANGULAR_CONFIG
-    }
-  ]
-})
-export class AnyModule {
-
-}
+    // ... other providers
+    provideRxRenderStrategies({
+      primaryStrategy: 'global',
+      patchZone: false,
+    }),
```

Comment:
looks like it would be a good idea to use `global`. Let's not raise this impression. 

```suggestion
    provideRxRenderStrategies({
      primaryStrategy: 'userBlocking',
      patchZone: false,
    }),
```